zproject/prod_settings_template.py

```diff
@@ -447,6 +447,16 @@
 EMAIL_GATEWAY_IMAP_FOLDER = "INBOX"
 
 
+########
+# Zoom integration.
+#
+# Zulip supports using Zoom as a video calling provider. To learn more about
+# configuring Zoom, see:
+# https://zulip.readthedocs.io/en/latest/production/settings.html#zoom-configuration
```

Comment:
This link is I think wrong, since it's on its own page?  Though maybe we want to move it, just noting this to make sure we don't forget.